service/history/execution/context_util.go

```diff
@@ -155,5 +155,15 @@ func emitWorkflowCompletionStats(
 			tag.WorkflowRunID(runID),
 			tag.WorkflowDomainName(domainName),
 		)
+	case types.EventTypeWorkflowExecutionContinuedAsNew:
+		scope.IncCounter(metrics.WorkflowContinuedAsNew)
+	default:
+		scope.IncCounter(metrics.WorkflowCompletedUnknownType)
+		logger.Info("NOT emitting completion stats for event type",
+			tag.WorkflowEventType(event.EventType.String()),
```

Comment:
We might just put this in the message string using `fmt.Sprintf`. Tags are mostly useful when we want to filter different type of logs.

Reply:
I had Emrah clarify his comment and I agree that constant strings are in general nicer for querying the "message" in logs. Ender, let me know if you feel strongly about using fmt.Sprintf instead of a tag and we can discuss more.